### DISCUSSION TITLE
Old props not being saved in properties column

### DISCUSSION THREAD

**Discussion by andreiurzica** [Z3d0X/filament-logger#40] [Q&A] [answered]
Hello,

Whenever I update a model, old properties are not being saved in the activity log. Is there an option to activate this feature?

Thank you!

**Comment by Z3d0X** [comment, marked as answer]
Disable logging from this package, via config file, and use 'logging-model-events' functionality provided by `spatie/laravel-activitylog`. 

https://spatie.be/docs/laravel-activitylog/v4/advanced-usage/logging-model-events

**Reply by dharen008** [reply]
Are you thinking about adding this as another option in the configuration file?

**Comment by fsamapoor** [comment]
@Z3d0X I was also confused in the beginning and after a deep dive into the source code, I found out that the package's logger structures the logs differently.
I would suggest adding this option and the default behavior of not structuring the log based on the `spatie/laravel-activitylog` in the README file.

**Comment by flashmediasolutions** [comment]
For anyone who finds this in an attempt to save the old + new props.

I extended the ResourceLogger and then added an altered update function that looks for changes and gets their corresponding original value for the log entry.

```
public function updated(Model $model)
    {
        $changes = $model->getChanges();

        // Ignore the changes to remember_token and updated_at
        if (count($changes) === 1 && (array_key_exists('remember_token', $changes) || array_key_exists('updated_at', $changes))) {
            return;
        }

        // Prepare attributes for log
        $attributes = collect($model->getOriginal())
            ->only(array_keys($changes))
            ->reject(fn ($value, $key) => $key === 'updated_at');

        $this->log(
            $model,
            'Updated',
             attributes:[
                'attributes' => collect($changes)->only($attributes->keys())->all(),
                'old' => $attributes->all(),
            ]
        );
    }
```

**Reply by dharen008** [reply]
tried and its working! thank you but If the plugin gets updated, it might stop working. I hope the developer thinks about adding this as a option in the configuration file.
